wright-filter-wheels/wright_filter_wheels/_yaqd_wright_filter_wheels_continuous.py

```diff
@@ -17,10 +17,13 @@ def __init__(self, name, config, config_filepath):
         super().__init__(name, config, config_filepath)
         self._motornum=config["motor"]
         self._serial_port = aserial.ASerial(config["serial_port"], config["baud_rate"])
-        self.microstep=1
+        self._microstep=config["microstep"]
+        self._units=config["units"]
+        self._set_microstep(self._microstep)
+        time.sleep(0.1)
         self._steps_per_rotation=400
-        self.home()
-
+        self._home()   # this will be removed once state TOML loads current position...may replace with home()
+        self._position=0            # "
```

Comment:
It should already load the current position (well, very first run will be NaN, so maybe check for that and then home and zero)